README.md

```diff
@@ -1,2 +1,12 @@
 # momkhulu
 Hospital resource allocation interface
+
+## Running
+
+1. Set up redis on docker port 
+
+sudo dockerrun -p 6379:6379 -d redis:2.8
+
+2. Run the server.
+
+python3 manage.py runserver
```

Comment:
look at the read me on this repository (https://github.com/praekelt/molo-gem)  for a guide on how we add readmes. You can make those changes in a separate PR